docs/development/code.md

```diff
@@ -37,3 +37,32 @@ without having to write boilerplate code.
 processing them in the background with workers.
 
 It will be used to process the jobs that will be created by the runner manager.
+
+## Typing
```

Comment:
Explain what this module is used for.

Reply:
Already explained, but I added a mini intro here that might make it more clear.